<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionExportWizard</class>
 <widget class="QWizard" name="SelectionExportWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::NoDefaultButton</set>
  </property>
  <widget class="QWizardPage" name="selectionPage">
   <property name="title">
    <string>Selection</string>
   </property>
   <property name="subTitle">
    <string>Select a book or items</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Project</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="optionPage">
   <property name="title">
    <string>Options</string>
   </property>
   <property name="subTitle">
    <string>Select options and the destination</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="extensionGroupBox">
      <property name="title">
       <string>Choose an extension</string>
      </property>
      <layout class="QHBoxLayout" name="formatButtonsLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="destinationWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="destinationLabel">
         <property name="text">
          <string>Destination:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="destinationLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="selectDestinationButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="fontSizeLabel">
        <property name="text">
         <string>Font size:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="fontSizeSpinBox">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fontFamilyLabel">
        <property name="text">
         <string>Font family:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFontComboBox" name="fontComboBox">
        <property name="currentFont">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="paragraphTopMarginLabel">
        <property name="text">
         <string>Paragraph top margin:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="paragraphTopMarginSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="paragraphFirstLineIndentLabel">
        <property name="text">
         <string>Paragraph's first line indent:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="paragraphFirstLineIndentSpinBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="spaceBetweenLinesLabel">
        <property name="text">
         <string>Space between lines</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spaceBetweenLinesSpinBox">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="minimum">
         <number>50</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
